File: 15_이준용/11047.py
from sys import stdin

n, k = map(int, stdin.readline().split())
coins = []
coin_count = 0

for _ in range(n):
    coins.append(int(stdin.readline()))

coins.reverse()
for coin in coins:
    if k >= coin:
        count, k = divmod(k,coin)
        coin_count += count
print(coin_count)

File: 22_최현진/11047.c
#include <stdio.h>

int main(void) {
    int n, k, i, coin[11], count = 0;
    scanf("%d %d", &n, &k);
    for (i = 0; i < n; i++) {
        scanf("%d", &coin[i]);
    }
    for (i = n - 1; i >= 0; i--) {
        count += (k / coin[i]);
        k %= coin[i];
    }
    printf("%d", count);
    return 0;
}